client/src/containers/QuestionsAnswers.jsx

```diff
@@ -40,20 +43,28 @@ const QuestionsAnswers = ( {itemId} ) => {
 
   useEffect(() => {
     setResultsToShow(questionData.slice(0, currentCount))
-  }, [questionData])
+    console.log('quesitonData', questionData)
+  }, [questionData, currentCount])
 
 
-  const handleClick = () => {
+  const handleLoadMoreQuestions = () => {
+    console.log('in load more answers::::', currentCount)
```

Comment:
please remove this conosle.log and the one on the following line

Reply:
updated